design/src/rfcs/rfc0034_generating_reusable_code.md

```diff
@@ -0,0 +1,232 @@
+# RFC: Principles for Generating Reusable Code
+
+> Status: Accepted
```

Comment:
On a previous comment: @LukeMathWalker I'd be happy to accept for merge interesting RFCs, regardless of whether they're accepted or not

Reply:
I disagree - we can merge an RFC either as `Accepted` or `Rejected`. Why would we merge an RFC without a clear outcome? 🤔

Reply:
Sorry. That's what I mean. Update the "status" field before merging to either "Accepted" or "Rejected" depending on the decision. My point is to start as `Status: RFC`

Reply:
Either works to be honest, one can start optimistic and then change to `Rejected` before merging 😛 I have no strong opinion there.